website/data/editors.yml

```diff
@@ -1,35 +1,35 @@
 - image: "/images/editors/atom-128px.png"
   name: Atom
-  content: >
-    [`prettier-atom`](https://github.com/prettier/prettier-atom),
+  content: |
+    [`prettier-atom`](https://github.com/prettier/prettier-atom)
```

Comment:
@j-f1 This won't put them on their own lines, right? (Would need double line break)

Reply:
The `>` on line 3 collapses single newlines, but changing it to a `|` preserves newlines.